packages/lenovolegionlinux-dkms-git/lenovolegionlinux-dkms-git.pacscript

```diff
@@ -0,0 +1,65 @@
+# Maintainer: MrDuartePT <gonegrier.duarte@gmail.com>
+
+_pkgname=lenovolegionlinux
+pkgname=${_pkgname}-dkms-git
+pkgver=1.c1e9b55
+pkgrel=0
+pkgdesc="LenovoLegionLinux (LLL) DKMS module"
+arch=("x86_64")
+url="https://github.com/johnfanv2/LenovoLegionLinux"
+license=('GPL-2.0-only')
+makedepends=(
+  "git"
+  "lm-sensors"
+  "i2c-tools"
+  "dmidecode"
+)
+depends=(
+  "dkms"
+  "${_pkgname}-git"
```

Comment:
```suggestion
```

This doesn't exist in apt repos.

Reply:
That ideia is to depend is lenovolegionlinux-git since have the userspace tools to interact with the driver.

Reply:
Then it should be a `pacdep`, not a regular `depends`.